code/workspaces/client-api/src/dataaccess/dataStorageRepository.js

```diff
@@ -40,19 +40,19 @@ function deleteByName({ sub }, name) {
   return DataStorage().remove(query).exec();
 }
 
-function update({ sub }, name, updatedValues) {
+function update(_, name, updatedValues) {
   const updateObj = {
     $set: updatedValues,
   };
   return DataStorage().findOneAndUpdate({ name }, updateObj, { upsert: false });
 }
 
-function addUsers({ sub }, name, userIds) {
+function addUsers(_, name, userIds) { // user, name, userId
   const updateObj = setUsers(userIds);
   return DataStorage().findOneAndUpdate({ name }, updateObj, { upsert: false });
 }
 
-function removeUsers({ sub }, name, userIds) {
+function removeUsers(_, name, userIds) { // user, name, userId
```

Comment:
How come these have all be replaced with a `_` argument? Does this have a special meaning?

Reply:
change in the linting rules prohibits unused variables, hence we are using `_` as a placeholder. The comment was to add clarification as to the ordering of the arguments.